notification/package.json

```diff
@@ -45,7 +45,8 @@
     "bunyan": "^1.8.12",
     "lodash": "^4.17.15",
     "node-fetch": "^2.6.0",
-    "serialize-error": "^6.0.0"
+    "serialize-error": "^6.0.0",
+    "@adyen/api-library": "^3.1.3"
```

Comment:
Lil' thing - usually we sort those dependencies, so this `@adyen/api-library` should be moved to the top.

Reply:
I updated it in: https://github.com/commercetools/commercetools-adyen-integration/pull/284